src/main/java/org/zalando/problem/spring/web/advice/AdviceTrait.java

```diff
@@ -79,7 +79,7 @@
  * @see RoutingAdviceTrait
  * @see ValidationAdviceTrait
  */
-public interface AdviceTrait {
+public interface AdviceTrait extends StatusConverter {
```

Comment:
So simply change `package` scope to `public` and let users to implements it if needed? And no implements of `StatusConverter` inside _problem-spring-web_ module